docs/auth.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+kcp implements the RBAC based authorization scheme as Kubernetes.
+Other authorization schemes (i.e. ABAC) are currently not supported.
+Generally, the same (cluster) role and (cluster) role binding priniciples apply the same way as in Kubernetes.
+
+However, since kcp is based around workspaces and workspaces are forming a tenancy boundary,
+RBAC rules are always declared within the context of a concrete workspace.
+These rules apply within the workspace they were declared and do not apply in other workspaces.
```

Comment:
Rules created in a workspace apply to requests against this workspace, independently from the workspace type. This is what resemembles Kubernetes authorization model.

In addition, rules can play a role in authorization to requests to other workspaces. The details will be described below.

Reply:
great suggestion, i took it 1:1 👍